pdr_backend/predictoor/predictoor_agent.py

```diff
@@ -118,6 +120,21 @@ def take_step(self):
         self.prev_block_number = UnixTimeS(self.cur_block_number)
         self.prev_block_timestamp = UnixTimeS(self.cur_timestamp)
 
+        # within the time window to run payout?
+        if (
+            self.s_start_payouts != 0
+            and self.cur_epoch_s_left < self.s_start_payouts
+            and self.cur_epoch not in self.prev_submit_payouts
+        ):
+            # run payout
+            # pylint: disable=line-too-long
+            logger.info(
+                "Running payouts, set predictoor_ss.bot_only.s_start_payouts to 0 to disable auto payouts"
+            )
```

Comment:
The log doesn't need to give instructions on how to do / not do certain things (eg disable auto payouts). 

Make the log note short enough such that it fits on one line. Example:
```python
logger.info('Running payouts...`)
```

Which means you can also remove `# pylint: disable=line-too-long`.

The result: 4 lines becomes 1 :)